src/Files.App/Filesystem/Cloud/Detector/SynologyDriveCloudDetector.cs

```diff
@@ -3,32 +3,35 @@
 
 using Files.Shared.Cloud;
 using Microsoft.Data.Sqlite;
-using System;
-using System.Collections.Generic;
 using System.IO;
 using Windows.Storage;
 
 namespace Files.App.Filesystem.Cloud
 {
+	/// <summary>
+	/// Provides an utility for Synology Drive Cloud detection.
+	/// </summary>
 	public class SynologyDriveCloudDetector : AbstractCloudDetector
 	{
 		protected override async IAsyncEnumerable<ICloudProvider> GetProviders()
 		{
-			/* Synology Drive stores its information on some files, but we only need sys.sqlite, which is placed on %LocalAppData%\SynologyDrive\data\db
-             * In this database we need "connection_table" and "session_table" tables:
-             * connection_table has the ids of each connection in the field "id", and the type of connection in the field "conn_type" (1 for sync tasks and 2 for backups)
-             * Also it has "host_name" field where it's placed the name of each server.
-             * session_table has the next fields:
-             * "conn_id", which has the id that we check on connection_table to see if it's a sync or backup task
-             * "remote_path", which has the server folder. Currently it's not needed, just adding in case in the future is needed.
-             * "sync_folder", which has the local folder to sync.
-            */
+			// Synology Drive stores its information on some files, but we only need sys.sqlite, which is placed on % LocalAppData %\SynologyDrive\data\db
+			// In this database we need "connection_table" and "session_table" tables:
+			// connection_table has the ids of each connection in the field "id", and the type of connection in the field "conn_type"(1 for sync tasks and 2 for backups)
+			// Also it has "host_name" field where it's placed the name of each server.
+			// session_table has the next fields:
+			// "conn_id", which has the id that we check on connection_table to see if it's a sync or backup task
+			// "remote_path", which has the server folder.Currently it's not needed, just adding in case in the future is needed.
```

Comment:
```suggestion
			// connection_table has the ids of each connection in the field "id", and the type of connection in the field "conn_type" (1 for sync tasks and 2 for backups)
			// Also it has "host_name" field where it's placed the name of each server.
			// session_table has the next fields:
			// "conn_id", which has the id that we check on connection_table to see if it's a sync or backup task
			// "remote_path", which has the server folder. Currently it's not needed, just adding in case in the future is needed.
```

Reply:
Fixed